app/internal/page/certificate_provider_address.go

```diff
@@ -64,7 +65,13 @@ func CertificateProviderAddress(logger Logger, tmpl template.Template, addressCl
 				addresses, err := addressClient.LookupPostcode(r.Context(), data.Form.LookupPostcode)
 				if err != nil {
 					logger.Print(err)
-					data.Errors.Add("lookup-postcode", validation.CustomError{Label: "couldNotLookupPostcode"})
+
+					if errors.As(err, &place.InvalidPostcodeError{}) {
```

Comment:
since you aren't using the unwrapped error these could probably use `errors.Is` instead

Reply:
I tried this initially (and have just tried again now in case I got `target` and `err` the wrong way round) but `Is()` doesn't seem to recognise `err` as a `place.InvalidPostcodeError`. 

The whole piece on unwrapping errors confused me - is this to try to get down to a base type somehow?

Reply:
You'd need to change the error from being a struct to just a value to use `Is`, which would be fine since you aren't using the Message or Statuscode. But yeah, more change so don't know if you care about doing it